obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -1141,7 +1141,17 @@ bool OBS_API::openAllModules(int& video_err)
 #endif
 
 			obs_module_t* module;
-			int           result = obs_open_module(&module, plugin_path.c_str(), plugin_data_path.c_str());
+			int           result;
```

Comment:
it is better to init `result` with some default value. as if we catch exception no value be assigned to `result`.

Reply:
Added initialization values for a few variables!